lms/templates/instructor/instructor_dashboard_2/discussions.underscore

```diff
@@ -0,0 +1,5 @@
+<!-- Discussion Topics. -->
+<div class="discussions-nav" id="discussions-management" tabindex="-1">
+    <div class="cohort-course-wide-discussions-nav"></div>
```

Comment:
Let's change these classnames. I think just "course-wide-discussions-nav" and "inline-discussions-nav" would be fine.